book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I think it would be good to have `:rentals` as an `attr_accesor` as well, because you want to be able to create rentals for your books, and with the `attr_reader` you can only read, so please apply the necessary changes.